fastlane/Fastfile

```diff
@@ -121,65 +118,17 @@ lane :update_hybrid_common_versions do |options|
 
   if dry_run
     UI.message("ℹ️  Nothing more to do, dry_run: true")
-  else
-    replace_text_in_files(
-      previous_text: previous_version_number,
-      new_text: new_version_number,
-      paths_of_files_to_update: files_to_update
-    )
-  end
-end
-
-desc "Build hybrid example"
-lane :build_hybrid_example do |options|
-  options[:previous_pod_version] = parse_pod_version
-  # If we chain lanes together, we'll need to make sure we don't chdir out of the correct directory
-  Dir.chdir(get_root_folder) do
-    sh("yarn example")
-  end
-  Dir.chdir(File.expand_path("examples/purchaseTesterTypescript/ios", get_root_folder)) do
-    sh("pod install --repo-update")
-    sh("pod update PurchasesHybridCommon --repo-update")
-  end
```

Comment:
Right i wonder if we have a test to make sure the example app builds. That's the only thing I would consider useful here

Reply:
oh yes, we do. The `android` and `ios` jobs build the sample